components/brave_wallet/browser/swap_response_parser_unittest.cc

```diff
@@ -325,6 +326,39 @@ TEST(SwapResponseParserUnitTest, ParseJupiterQuote) {
   ASSERT_FALSE(swap_quote);
 }
 
+TEST(SwapResponseParserUnitTest, ParseJupiterQuoteSlippageBps) {
+  auto* json_fmt(R"(
+    {
+      "data": [
+        {
+          "inAmount": "10000",
+          "outAmount": "261273",
+          "amount": "10000",
+          "otherAmountThreshold": "258660",
+          "swapMode": "ExactIn",
+          "priceImpactPct": "1.1",
+          "slippageBps": %s,
+          "marketInfos": []
+        }
+      ],
+      "timeTaken": "0.044471802000089156"
+    })");
+
+  // OK: valid slippageBps value
+  std::string json = base::StringPrintf(json_fmt, "\"50\"");
+  mojom::JupiterQuotePtr swap_quote = ParseJupiterQuote(ParseJson(json));
+  ASSERT_TRUE(swap_quote);
+  EXPECT_EQ(swap_quote->routes.at(0)->slippage_bps, 50);
+
+  // KO: null slippageBps value
+  json = base::StringPrintf(json_fmt, "null");
+  ASSERT_FALSE(ParseJupiterQuote(ParseJson(json)));
```

Comment:
nit: usually `ASSERT_SMTH` is used when further test doesn't make sense if assertion fails. For example `ASSERT_TRUE(swap_quote)` is ok to be an assert as otherwise test crashes in the next line. 
`ASSERT_FALSE(ParseJupiterQuote(ParseJson(json)));` - In this case it is ok to proceed with test when parsing somehow succeeds, so should EXPECT_FALSE